spec/std/log/context_spec.cr

```diff
@@ -37,6 +38,53 @@ describe Log::Context do
     c({a: 1, b: 3}).merge(c({b: 2})).should eq(c({a: 1, b: 2}))
   end
 
+  describe "#empty?" do
+    it Number do
+      c(17).empty?.should be_false
+    end
+
+    describe Array do
+      it "empty" do
+        c([] of Int32).empty?.should be_true
+      end
+
+      it "not empty" do
+        c([1, 2, 3]).empty?.should be_false
+      end
+    end
+
+    describe String do
+      it "not empty" do
+        c("").empty?.should be_false
```

Comment:
Should this be true? 🤔

Reply:
In this case no because `#empty?` here is checking if there is _ANY_ value stored within the context.  Not if the underlying value is empty.

Reply:
So empty array and empty hash are not values...?

Reply:
Hmm, yea I guess it could be `true` since an empty string would essentially be `nil`, or the same as if a value was not provided.  Also it's probably not a common use case so, I'll update that.

Reply:
Nevermind `blank?`, I misread

Reply:
I updated it to consider `""` as empty.  I also ran into https://play.crystal-lang.org/#/r/8wgb 🤔.

Reply:
It's already reported as a bug where you can't more than two types in `case`. I would put each type in a different `when`, even if the code is the same.